src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/App/index.js

```diff
@@ -48,7 +48,7 @@ App.propTypes = {
 const mapStateToProps = state => ({
   authenticatedWithSSOe: isAuthenticatedWithSSOe(state),
   ehrDataByVhaId: selectEhrDataByVhaId(state),
-  facilities: selectPatientFacilitiesDsot(state),
+  facilities: selectPatientFacilities(state),
```

Comment:
@ryguyk before I finish reaching out to teams about this, can you help me understand one more thing?  I had thought we said: we are removing `selectPatientFacilities` so should have no references to it anywhere in the code anymore. However: in this change, we used to have `selectPatientFacilitiesDsot` function, now being renamed here to `selectPatientFacilities`.  Are these references of our own also now wrong? 

I'd like to make sure I'm clear for other teams who reference `selectPatientFacilities` - what is it I am actually asking them to do?

Reply:
It's not that `selectPatientFacilities` should never appear in our codebase. It will. The new selector and the old selector are _both_ named that. The real point here is to ensure that there is no code trying to import (and call) the _old_ one.

In the example above, the call to `selectPatientFacilities(state)` on line 51 is a call to the _new_ selector, which is imported on line 7 from the `cerner-dsot` path. 

As an extra point of clarity, this diff highlights that line 7 is slightly changed. It was already importing the new selector (which, again, is named `selectPatientFacilities` just like the old one), but it was previously creating an alias `selectPatientFacilitiesDsot` so that it could be differentiated against the old `selectPatientFacilities` when we were conditionally calling each of them based on our Flipper.